nixos/tests/systemd-networkd-nftset.nix

```diff
@@ -0,0 +1,132 @@
+# This tests systemd-networkd NFTSet option. The interface's statically
+# configured address is added to an nft set, and the DHCP configured address is
+# added to another. The sets are used by one rule that blocks connections to
+# the static address, and one rule that blocks connections to the DHCP address.
+# It is tested that the expected connections succeed or fail from another host.
+import ./make-test-python.nix (
```

Comment:
The manual specifies a different way of defining tests, based on `runTest` instead of importing this.
See https://nixos.org/manual/nixos/stable/#sec-call-nixos-test-in-nixos